WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuViewModel.swift

```diff
@@ -70,6 +70,18 @@ final class HubMenuViewModel: ObservableObject {
     private let cardPresentPaymentsOnboarding: CardPresentPaymentsOnboardingUseCaseProtocol
     private let posEligibilityChecker: POSEligibilityCheckerProtocol
 
+    // TODO:
+    // Is this the right place to instantiate the product provider and use property injection?
```

Comment:
I think this is good for now, we can revisit how we pass the view model to the entry point view later. If possible, maybe this variable can be marked as `private(set)`.

Reply:
Thanks, revisiting when clarifying DI makes sense to me. Also updated access control on https://github.com/woocommerce/woocommerce-ios/pull/12882/commits/9b988f09860e9ff963755383aa962ac9f8b53f6c